### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Uthmanbello**
## Highlights
- Deleted boilerplate in the Create React App
- Created a new directory containing Calculator.js and Calculator.css files
- Created a calculator design using JSX and CSS
- Exported calculator function and ensured it displayed well in the browser

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +0,0 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
```
Comment (omar-muhamad) [inline src/App.js]: - [x] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=1%20child%20component-,Once%20the%20Calculator%20is%20ready%2C%20import%20it%20and%20use%20it%20in%20your%20top%2Dlevel%20component%3A%20App.js,-Test%20it%20in), The App component suppose to be your top-level component and has Calculator component imported and used in it. Please restore the App component and include the Calculator component in it.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+function Calculator() {
+  return (
+    <div className="Calc">
+      <div className="row1 Display">0</div>
+      <div className="row2">
+        <button type="button" className="Calc-button">AC</button>
+        <button type="button" className="Calc-button">+/-</button>
+        <button type="button" className="Calc-button">%</button>
+        <button type="button" className="Calc-button Calc-operator">/</button>
+      </div>
```
Comment (omar-muhamad) [inline src/components/Calculator.js]: - [x] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Calculator.js%20file.-,In%20Calculator.js%2C%20create%20a%20React%20component%20that%20matches%20this%20design%3A,-Implement%20the%20design), Please replace the divide symbol you added `/` with this symbol `÷` instead to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Uthmanbello,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=now%20you%20should%3A-,Delete%20all%20the%20boilerplate%20from%20Create%20React%20App%20(text%2C%20images%2C%20styles).,-Create%20a%20new), Please remove all boilerplate files marked at the image below.

image

- [x] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Uthmanbello** [comment]
Hi @Omar-Muhamad ,
Thanks a lot for the feedback

**Review by topeogunleye — APPROVED** [review]
### Hi @Uthmanbello, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Uthmanbello** [comment]
Hi @topeogunleye, 
Thanks a lot for the feedback and nice gif 😆

**Pull request merged**